packages/tools/locales/el_GR.po

```diff
@@ -47,7 +49,7 @@ msgstr "(%s)"
 
 #: packages/lib/SyncTargetNone.ts:16
 msgid "(None)"
-msgstr ""
+msgstr "(None)"
```

Comment:
Would that actually be translated to (None)?

Reply:
Thanks @laurent22, I missed that actually. It's fixed now.